services/movie.py

```diff
@@ -1,14 +1,19 @@
+from django.db import transaction
 from django.db.models import QuerySet
 
-from db.models import Movie
+from db.models import Movie, Genre, Actor
 
 
 def get_movies(
-    genres_ids: list[int] = None,
-    actors_ids: list[int] = None,
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None,
```

Comment:
The type hint `list[int]` is used here. This syntax is only compatible with Python 3.9 and above. For broader compatibility, consider using `List[int]` from the `typing` module.